<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>proe</class>
 <widget class="QMainWindow" name="proe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>proe</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>30</y>
      <width>151</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Dodaj obiekt</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>95</width>
       <height>65</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="PCButton">
        <property name="text">
         <string>Pc</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="LaptopButton_2">
        <property name="text">
         <string>Laptop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="Laptop_gamingButton">
        <property name="text">
         <string>Laptop gaming</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="Dodaj">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>170</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Dodaj</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>271</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>dodawanie</string>
    </property>
    <widget class="QLineEdit" name="karta_graficznain">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="motherboard_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>119</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>model</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="modelin">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="motherboard_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>119</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>karta graficzna</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>127</width>
       <height>72</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="motherboard">
        <property name="text">
         <string>motherboard</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Processor_benchmark">
        <property name="text">
         <string>Processor_ocena(int)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="User">
        <property name="text">
         <string>Uzytkownik</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>135</width>
       <height>74</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLineEdit" name="motherboardin"/>
      </item>
      <item>
       <widget class="QLineEdit" name="processor_in"/>
      </item>
      <item>
       <widget class="QLineEdit" name="user_in"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>170</y>
      <width>281</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>wyswietlanie</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>121</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="motherboard_2">
        <property name="text">
         <string>motherboard</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Processor_benchmark_2">
        <property name="text">
         <string>Processor_benchmark</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="User_2">
        <property name="text">
         <string>Uzytkownik</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>135</width>
       <height>74</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLineEdit" name="motherboardout"/>
      </item>
      <item>
       <widget class="QLineEdit" name="processorout"/>
      </item>
      <item>
       <widget class="QLineEdit" name="userout"/>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="motherboard_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>119</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>karta graficzna</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="karta_graficznaout">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="motherboard_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>119</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>model</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="modelout">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>130</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>220</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prevButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>220</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Prev</string>
    </property>
   </widget>
   <widget class="QPushButton" name="usunButton">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>320</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>usun</string>
    </property>
   </widget>
   <widget class="QLabel" name="zdj">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>221</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>961</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProjekt3">
    <property name="title">
     <string>Dodaj</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuProjekt3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
